src/containers/IdentityContainer.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from 'react';
+
+import { esriPromise } from 'react-arcgis';
+
+import { Identity } from '../components/Identity';
+
+interface IdentityComponentProps {
+  credential?: __esri.Credential;
+  portalUrl: string;
+  handleLoginChange: (credential?: __esri.Credential) => void;
+}
+interface IdentityComponentState {}
+
+export class IdentityContainer extends React.Component<IdentityComponentProps, IdentityComponentState> {
+  identityManager: __esri.IdentityManager;
+
+  constructor(props: IdentityComponentProps) {
+    super(props);
+    this.handleLogin = this.handleLogin.bind(this);
+    this.handleLogout = this.handleLogout.bind(this);
+  }
+
+  componentDidMount() {
+    esriPromise([
+      'esri/identity/OAuthInfo',
+      'esri/identity/IdentityManager'
+    ]).then(([OAuthInfo, IdentityManager]) => {
+      this.identityManager = IdentityManager;
+
+      const info = new OAuthInfo({
+          appId: 'y4Lx1l6456Mbf85z',
+          popup: false
```

Comment:
Is there a reason that we're doing `popup: false`?  Avoiding reloading the app would be ideal although I'm unsure if that's possible...

Reply:
`popup: true` opens the login screen in a popup and the redirects to the app inside that popup. The process of reloading the app is the same it just happens in a different window.